### PR TITLE
Order logic refactoring alpha version

### PR DESCRIPTION
**Pull request by ArtemSobolenko**
When we add or delete orderItem balance calculating correctly in component table.
But I'm need do it with invoiceItem.

### CODE REVIEW

**Review by deivur — APPROVED** [review]

File: src/main/java/com/maybe/maybe/entity/InvoiceItem.java [hunk 1]
```diff
@@ -23,11 +23,11 @@
     private Component component;
 
     @NotNull
-    @Column(name = "quantity", nullable = false, scale = 15, precision = 3)
+    @Column(name = "quantity", nullable = false, scale = 15, precision = 5)
```
Comment (kosten27) [inline src/main/java/com/maybe/maybe/entity/InvoiceItem.java]: Why did you change precision in quantity?

**Review by CaH4aZzz — APPROVED** [review]

**Review by kosten27 — APPROVED** [review]

**Review by Klitsenko — APPROVED** [review]
